src/scenarios/shared/postcommands.py

```diff
@@ -3,5 +3,6 @@
 
 def clean_directories():
     to_remove = (APPDIR, TMPDIR, TRACEDIR, PUBDIR, BINDIR)
+    print(f"Removing {','.join(to_remove)} if exist ...")
```

Comment:
nit: Maybe move this in to the loop so we only print the folders we are removing

Reply:
That would be ideal, but whether the folder's removed is unkown in the loop...